notes.php

```diff
@@ -65,9 +65,9 @@
             <table class="table table-striped" style = "margin-top: -10px">
               <thead>
                 <tr>
+                    <th>ID</th>
```

Comment:
https://gyazo.com/a4c108a830574a787a26cbc47e89b904 

Must be PHPStorm,  didn't press tab at all >_>